### ISSUE TITLE
Add support for mixed fonts

### ISSUE THREAD

**Issue by gunchleoc** [ValyriaTear/ValyriaTear#553]
At the moment, Japanese can't use its native language name in the language selection menu, because we don't have support for mixed fonts.

Partial translations will probably also look better with mixed fonts.

I'm interested in taking on this bug myself once I have figured out my text truncation problem.

**Comment by Bertram25** [comment]
Yep, that would be an interesting feature. Please keep in mind that I would also like to be able later to add custom glyphs, so that we could insert images in the middle of the text. Like joystick buttons and such things. Just in case.

**Comment by gunchleoc** [comment]
If you create a markup language for that, the images could be referenced by printf placeholders, so that the translator won't touch the code, but still gets a proper sentence structure. So, that feature would be totally independent of the fonts, unless we use emojis.

**Comment by Bertram25** [comment]
We wouldn't use emojis, so yes, indeed. :)
